x-pack/test/functional/apps/ingest_pipelines/ingest_pipelines.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default ({ getPageObjects, getService }: FtrProviderContext) => {
+  const pageObjects = getPageObjects(['common', 'ingestPipelines']);
+  const log = getService('log');
+
+  describe('Ingest Pipelines', function() {
+    this.tags('smoke');
```

Comment:
FYI, this comments is long after this PR was merged, but I don't think this `this.tags('smoke');` does anything.  I think the correct usage is `this.tags(['smoke']);`.
There are some jobs that only run smoke tests and it doesn't run this one.  In fact, there's currently only 1 smoke test that runs with packaging tests.